default.py

```diff
@@ -103,7 +113,7 @@ def POPDUBBED_PAGES(payload):
 
 @route('search')
 def SEARCH(payload):
-    query = control.keyboard(control.lang(30007))
+    query = control.keyboard(control.lang(30002))
```

Comment:
please revert this change, 30007 is the correct value here.
it is refering to "Search" string:
https://github.com/DxCx/plugin.video.9anime/blob/c8a2c974f1eee1527624eb567ce329c71ee693d1/resources/language/English/strings.xml#L13

Reply:
I had not done any changes in search function.
It may have got from copy/pasting from another files I guess.
Will change it.